src/Jackalope/Transport/TransportInterface.php

```diff
@@ -319,12 +324,11 @@ public function setNodeTypeManager($nodeTypeManager);
      * @param array string names of node types to fetch, if empty array all
      *      node types are retrieved
      *
-     * @return array with the definitions (see
-     *      Jackalope\NodeTypeDefinition::fromArray for what is expected)
+     * @return array with the definitions
      *
-     * @see Jackalope\NodeTypeDefinition::fromArray
+     * @see NodeTypeDefinition::fromArray
      */
-    public function getNodeTypes($nodeTypes = array());
+    public function getNodeTypes(array $nodeTypes = array());
```

Comment:
this is a BC break. please rebase on the upstream master, where i reverted this.